src/unittests/platform/OSXKeyStateTests.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Deskflow -- mouse and keyboard sharing utility
+ * SPDX-FileCopyrightText: (C) 2025 Chris Rizzitello <sithlord48@gmail.com>
+ * SPDX-License-Identifier: GPL-2.0-only WITH LicenseRef-OpenSSL-Exception
+ */
+#include "base/Log.h"
+
+#include "../../lib/platform/OSXKeyState.h"
```

Comment:
I'm confused why you're mixing relative and absolute file paths.

I think they should either all be relative or or all absolute. I prefer absolute so you don't have to waste time figuring out how many dirs to traverse up.

```suggestion
#include "base/Log.h"
#include "platform/OSXKeyState.h"
```

Reply:
Relative paths are the classes the test is testing. 
"Absolute"  are the additional classes needed to support the tests.